Firestore/Source/Public/FirebaseFirestore/FIRTimestamp.h

```diff
@@ -33,8 +33,8 @@ NS_ASSUME_NONNULL_BEGIN
 NS_SWIFT_NAME(Timestamp)
 @interface FIRTimestamp : NSObject <NSCopying>
 
-/** :nodoc: */
-- (instancetype)init NS_UNAVAILABLE;
+/** Initialiser. Only for use in unit tests. */
+- (instancetype)init;
```

Comment:
I think we should just make the designated initializer public here:
https://github.com/firebase/firebase-ios-sdk/blob/627ebdcd4575d8f4ca7b7942fe183a54ec0b2ab3/Firestore/Source/API/FIRTimestamp.m#L68

Reply:
We might not even need this. Rolling back for now.